### ISSUE TITLE
Better GPU mapping

### ISSUE THREAD

**Issue by guilbaults** [guilbaults/slurm-job-exporter#2]
The env variable SLURM_JOB_GPUS is used to map the GPUs to a job. 

This does not work with slurm_adopt. Ideally it should read the device whitelist in the cgroup, but I could not find how to read how the GPUs are binded to a job cgroup.

**Comment by guilbaults** [comment]
cgexec and nvidia-smi can be used to find the GPUs of a job
```cgexec -g devices:slurm/uid_3015160/job_402/ nvidia-smi --list-gpus
GPU 0: NVIDIA A100-SXM4-40GB (UUID: GPU-21a5757c-414f-8dd6-7fff-4a409de596f3)
  MIG 4g.20gb     Device  0: (UUID: MIG-4f421ec4-5175-50ef-8452-d6abb13cb1d2)```

**Comment by guilbaults** [comment]
Fixed in #15 , cgexec and nvidia-smi is now used
